core/server/src/main/java/alluxio/worker/netty/DataServerReadHandler.java

```diff
@@ -185,7 +186,7 @@ private boolean tooManyPendingPackets() {
   }
 
   /**
-   * Requires to hold mLock when calling this.
+   * Requires to hold mLock when calling this. This is only called in the event loop thread.
```

Comment:
I didn't know it could be used for method.  It looks it could be. http://jcip.net/annotations/doc/net/jcip/annotations/GuardedBy.html 

Done